satpy/composites/__init__.py

```diff
@@ -555,26 +555,29 @@ def _insert_palette_colors(channels, palette):
 class DayNightCompositor(GenericCompositor):
     """A compositor that blends a day data with night data."""
 
-    def __init__(self, name, lim_low=85., lim_high=88., **kwargs):
+    def __init__(self, name, lim_low=85., lim_high=88., chose_day_night="day_and_night", **kwargs):
         """Collect custom configuration values.
 
         Args:
             lim_low (float): lower limit of Sun zenith angle for the
                              blending of the given channels
             lim_high (float): upper limit of Sun zenith angle for the
                              blending of the given channels
+            chose_day_night (string): "day_and_night" means both day and night parts will be kept
+                                      "only_day" means only day part will be kept
+                                      "only_night" means only night part will be kept
 
         """
         self.lim_low = lim_low
         self.lim_high = lim_high
+        self.chose_day_night = chose_day_night
         super(DayNightCompositor, self).__init__(name, **kwargs)
 
     def __call__(self, projectables, **kwargs):
         """Generate the composite."""
         projectables = self.match_data_arrays(projectables)
 
-        day_data = projectables[0]
-        night_data = projectables[1]
+        data_1 = projectables[0]
```

Comment:
Is there a more descriptive name for this? I suppose "foreground" isn't really correct here.

Reply:
I think you are right. Whatever you choose, you need a 'foreground' image at least. So 'foreground' and 'background' are more suitable.